dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -110,6 +110,69 @@ def test_load_data_labeler(self, *mocks):
         data_labeler = dp.DataLabeler(labeler_type='unstructured')
         self.assertIsInstance(data_labeler, BaseDataLabeler)
 
+    def test_structured_data_labeler_fit_predict_take_data_obj(self):
```

Comment:
I'm concerned these are running the actual data labeler rather than validating that the function returns the proper formats and accepts the proper formats.

Reply:
This is a good point, I mostly had this at first because I didn't know if the labeler could work with several formats, and if I would return several, but since its canonical now I suppose I could write this without calls to fit if you think that would be more appropriate.